internal/apischema/openai/openai.go

```diff
@@ -581,7 +582,6 @@ const (
 	ChatCompletionChoicesFinishReasonLength        ChatCompletionChoicesFinishReason = "length"
 	ChatCompletionChoicesFinishReasonToolCalls     ChatCompletionChoicesFinishReason = "tool_calls"
 	ChatCompletionChoicesFinishReasonContentFilter ChatCompletionChoicesFinishReason = "content_filter"
-	ChatCompletionChoicesFinishReasonFunctionCall  ChatCompletionChoicesFinishReason = "function_call"
```

Comment:
why is this getting deleted

Reply:
It is not used anywhere in the code, we can add it if we need it in the future

Reply:
function_call is deprecated and replaced by tool_calls